parts/k8s/artifacts/kuberneteskubelet.service

```diff
@@ -30,7 +30,7 @@ ExecStart=/usr/bin/docker run \
   --volume=/etc/kubernetes/:/etc/kubernetes:ro \
   --volume=/srv/kubernetes/:/srv/kubernetes:ro $DOCKER_OPTS \
   --volume=/var/lib/waagent/ManagedIdentity-Settings:/var/lib/waagent/ManagedIdentity-Settings:ro \
-  --volume=/usr/libexec/kubernetes/kubelet-plugins:/usr/libexec/kubernetes/kubelet-plugins \
+  --volume=/usr/libexec/kubernetes/kubelet-plugins:/usr/libexec/kubernetes/kubelet-plugins:ro \
```

Comment:
Why do we need to modify Linux kubelet?

Reply:
@andyzhangx Is this a fix to the prior PR? Is a read-only volume preferable?

Reply:
@jackfrancis yes, it's better be read-only